tools/mason/MasonPublish.chpl

```diff
@@ -480,6 +481,17 @@ proc check(username : string, path : string, trueIfLocal : bool, ci : bool) thro
     writeln(spacer);
   }
 
+  if package {
+    writeln('Checking for license:');
+    if checkLicense(projectCheckHome) {
+      writeln('   Found valid license in manifest file. (PASSED)');
+    } else {
+      writeln('   Invalid license name. Please use a valid name from SPDX license list. (FAILED)');
```

Comment:
Not required, but it could be nice to include the name that was parsed to account for formatting errors, e.g.

```
Invalid license name: 'foo'
    Please use a valid name from SPDX license list. (FAILED)
```